server/routes/index.js

```diff
@@ -1,8 +1,23 @@
 import { Router } from 'express';
+import passport from 'passport';
 import userAuth from './api/auth/userAuth';
+import Facebook from '../controllers/auth/strategies/facebookStrategy';
 
-const router = Router();
 
+const router = Router();
 router.use('/api/v1', userAuth);
+// router.use('/api', api);
+router.get('/api/v1/facebook',
+  passport.authenticate('facebook',
+    { session: false }), (req, res) => {
+      res.json({
+        msg: 'Works'
+      });
+});
```

Comment:
Please remove the inline code comment 
Also I feel like the code ``` passport.authenticate('facebook',
    { session: false }), (req, res) => {
      res.json({
        msg: 'Works'
      }); ```
should be in the controller and imported into the router. 
Router takes in url and determines which part to go(most likely controller or middleware)

Reply:
Okay